src/coreclr/nativeaot/BuildIntegration/Microsoft.NETCore.Native.targets

```diff
@@ -34,7 +34,6 @@ The .NET Foundation licenses this file to you under the MIT license.
 
   <!-- Set up the defaults for the compatibility mode -->
```

Comment:
We do not have a thing called compatibility mode anymore. Delete this PropertyGround and change the IlcScanReflection condition polarity to imply the default?